### PR TITLE
[Installer] Add support for default plugins

### PR DESCRIPTION
**Pull request by segiddins**
\c @orta

### CODE REVIEW

**Comment by segiddins** [comment]
@orta that CI failure includes an exception in stats

**Comment by orta** [comment]
Thanks, will take a look

**Comment by orta** [comment]
alright, I have a sense of this, will see if I can take over this PR today/tomorrow

**Comment by segiddins** [comment]
@orta, @kylef  this should be good to go, please review. I'll update the integration specs when I merge.

File: lib/cocoapods/installer.rb [hunk 1]
```diff
@@ -453,6 +453,17 @@ def ensure_plugins_are_installed!
       end
     end
 
+    DEFAULT_PLUGINS = { 'cocoapods-stats' => {} }
```
Comment (kylef) [inline lib/cocoapods/installer.rb]: Could we get an explicit test to check that the stats plugin is enabled by default? And also to ensure its gone when the disable stats end variable is set (unless it's handled elsewhere, but I feel this is the place).

Comment (segiddins) [inline lib/cocoapods/installer.rb]: nope, it's disabled inside of stats itself

Comment (segiddins) [inline lib/cocoapods/installer.rb]: (and that test is handled by the integration specs, no?)

**Comment by segiddins** [comment]
Will merge when green.

**Pull request merged**